test/Chainweb/Test/Pact/PactMultiChainTest.hs

```diff
@@ -233,6 +234,42 @@ chainweb213Test = do
         , "(create-table tbl)"
         ]
 
+pactLocalDepthTest :: PactTestM ()
```

Comment:
Maybe there should be more testing scenarios? When the depth is too big or negative for example? What do you think?

Reply:
I think that's a great idea.

Reply:
@ak3n Let's make sure we have coverage for negative rewind depths, and consult @larskuhtz on what a reasonable maximum rewind depth would be good for the nodes. I would assume we don't want people walking all the way back to genesis.

Reply:
In particular we have to test interactions with the different rewind limits that we have in place.

Generally, `local` is used via the service API, so we can be generous. We should give the user the ability to configure that, because it's their call how much load they expect and how much they are willing to compromise and latencies for other pact requires.

In practice, I expect that more than a few hundred blocks (also depending on the chain) would probably cause significant slow down. So, we may either put a limit or warn the user if they configure a larger number.

Reply:
This is easy to test, because we log execution time traces for all pact calls. So we can run a node with a large limit and just plot latencies pact request latencies (also of other pact requests) over the requested confirmation depth.